features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/themes/wso2/templates/life-cycles/template.jag

```diff
@@ -87,7 +88,52 @@ jagg.message({
          $(".modal-footer").html("");
     }
     });
-}else{
+} else if (isSetDeprecation === 'true') {
+
+    var messageContent = i18n.t("All the previous versions of the API will be deprecated. Continue?");
+    var messageTitle = i18n.t("Change Lifecycle status");
+
+    jagg.message({
+        content: messageContent,
+        type:"confirm",
+        title: messageTitle,
+        anotherDialog:true,
+        okCallback:function() {
+            $('#messageModal').modal({backdrop: 'static', keyboard: false });
+            $(".modal-header .close").hide();
+            $(".modal-footer").html("");
+            $(".modal-title").html(i18n.t("Please wait"));
+            $(".modal-body").addClass("loadingButton");
+            $(".modal-body").css({"margin-left":25}); //todo param_string
+            $(".modal-body").html(i18n.t("Changing the API lifecycle status") + " : " + apiName + " - " + version );
+
+            jagg.post("/site/blocks/life-cycles/ajax/life-cycles.jag", {
+                   action: "updateStatus",
+                   provider: provider,
+                   name: apiName,
+                   version: version,
+                   status: status
+            }, function (result) {
+                if (!result.error) {
+                    $('#statusUpdateMsg').show();
+                    setTimeout(function(){
```

Comment:
Fixed in https://github.com/wso2/carbon-apimgt/commit/e3c49af10e8752b1b6531c3fabea1c1c56e1330d